app/views/admin/projects/show.html.erb

```diff
@@ -1,23 +1,23 @@
 <h3><%= @project.name %></h3>
 <ul>
-  <li><strong>Link to send students:</strong> <%= link_to new_project_clone_path(project_id: @project.hash_id), new_project_clone_path(project_id: @project.hash_id) %></li>
+  <li><strong>Link to send students:</strong> <%= link_to new_project_clone_url(project_id: @project.hash_id), new_project_clone_url(project_id: @project.hash_id) %></li>
```

Comment:
changed this to be the full URL so we can enforce the SSL link on these for students going forward; visiting conveyor belt on a non-SSL URL will block login from working